Source/ARTRealtime.m

```diff
@@ -778,6 +778,7 @@ - (void)onConnected:(ARTProtocolMessage *)message {
             // Reattach all channels with specified states
             for (ARTRealtimeChannelInternal *channel in self.channels.nosyncIterable) {
                 if (channel.state == ARTRealtimeChannelAttaching || channel.state == ARTRealtimeChannelAttached || channel.state == ARTRealtimeChannelSuspended) {
+                    channel.state = ARTRealtimeChannelAttaching;
```

Comment:
I don't think it should be altered directly, it's done later in a call to `[self transition:ARTRealtimeChannelAttaching status:status];`

Reply:
Is there a fix for this?